internal/ipxe/http/middleware.go

```diff
@@ -27,14 +27,16 @@ func (h *loggingMiddleware) ServeHTTP(w http.ResponseWriter, req *http.Request)
 	if uri == "/metrics" {
```

Comment:
Later when we get a chance, we should get rid of this logic and make metrics do the same thing that iso does right now. so there's uniformity.